src/plugins/apm_oss/server/index.ts

```diff
@@ -21,6 +21,7 @@ export const config = {
     sourcemapIndices: schema.string({ defaultValue: 'apm-*' }),
     onboardingIndices: schema.string({ defaultValue: 'apm-*' }),
     indexPattern: schema.string({ defaultValue: 'apm-*' }),
+    fleet: schema.boolean({ defaultValue: true }),
```

Comment:
Option to disable fleet support. Should it be called something else like `fleet.enabled`?